src/pushin.js

```diff
@@ -114,13 +194,9 @@ class pushIn {
     const { top } = this.scene.getBoundingClientRect();
 
     let inpoints = [top];
-    if (elem.dataset.hasOwnProperty('pushinFrom')) {
+    if (hasOwnProperty.call(elem.dataset, 'pushinFrom')) {
       inpoints = elem.dataset.pushinFrom.split(',');
-      inpoints = inpoints.map(inpoint => parseInt(inpoint.trim()));
-    } else if (i === 0 && this.scene.dataset.hasOwnProperty('pushinFrom')) {
```

Comment:
@arturovt  here is another example of a method being removed for some reason.

Reply:
Yeah, I didn't think it's used, but I saw you fixed it another PR. Lint told me that it's an unused variable.

Reply:
Got it, yeah it will be fixed in the next patch release. Thanks!